### PR TITLE
Infra 2191 dapple

### PR DESCRIPTION
**Pull request by nathanleiby**
Issue: https://clever.atlassian.net/browse/infra-2191

Add support for Dapple, allowing automated deploys (CD).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ $ ./circleci/catapult-publish [CATAPULT_URL] [CATAPULT_USER] [CATAPULT_PASS] [AP
 
 If you need to publish multiple applications, run this command once for each.
 
+### Dapple
```
Comment (mohit) [inline README.md]: Let's move this in a seperate section which is Clever only. Okay with leaking the name and URIs but don't think we can make dapple public yet. Let's make that explicit

File: circleci/dapple-deploy [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```
Comment (mohit) [inline circleci/dapple-deploy]: move this into a seperate directory or naming scheme that makes it clear that this is Clever internal.

File: README.md [hunk 3]
```diff
@@ -43,6 +43,17 @@ $ ./circleci/catapult-publish [CATAPULT_URL] [CATAPULT_USER] [CATAPULT_PASS] [AP
...
+
+Deploys your application with [dapple](https://github.com/clever/dapple).
+Requires that you've first pushed the Docker image and published the application to Catapult.
+
+```
+$ ./circleci/dapple-deploy [DAPPLE_URL] [DAPPLE_USER] [DAPPLE_PASS] [APP_NAME]
```
Comment (mohit) [inline README.md]: use `DAPPLE_TOKEN` rather than user/pass?

Comment (nathanleiby) [inline README.md]: This follows the pattern we're currently using with Catapult publish, so I want to keep them parallel.

**Pull request merged**